app/views/student/show.html.erb

```diff
@@ -1,6 +1,21 @@
-<br/>
+<h1>Student Profile Page: ID <%= params[:id] %> </h1>
```

Comment:
Please reset this file to it's old state. The profile of the mentee has been perfected, and the changes you have proposed here will only regress this back to the way it was before my last set of changes.

It is the three column profile page, with news feed and everything else, built in.